auth/standard_role_manager.cc

```diff
@@ -340,7 +351,6 @@ future<> standard_role_manager::drop(std::string_view role_name) {
             static const sstring query = format("SELECT member FROM {}.{} WHERE role = ?",
```

Comment:
We will probably face similar concerns about the maximum mutation size in the `standard_role_manager::drop` method as we did with `default_authorizer::revoke_all`, because `modify_membership` might potentially be called a large number of times...

Reply:
But not in a single transaction, there are couple cases with complex code which I am not planning (yet) to refactor to use single transaction. After-all consistency would not be worse than in current situation. Will it still fail?

Reply:
No, if the mutations are of the right schema and size then I don't see a reason why applying them should fail.